.ci/operatorhub.sh

```diff
@@ -34,7 +35,9 @@ git rebase -q upstream/master
 
 for dest in upstream-community-operators community-operators; do
     cp "${OLD_PWD}/${PKG_FILE}" "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${DEST_PKG_FILE}"
-    cp "${OLD_PWD}/${CSV_FILE}" "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${DEST_CSV_FILE}"
+    mkdir "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${VERSION}"
```

Comment:
Could you add a `-p` to the command? I think it might fail if the directory exists already, and won't fail if `-p` is passed.